docs/pages/access-controls/login-rules/guide.mdx

```diff
@@ -17,13 +17,38 @@ first Login Rule to your Teleport cluster.
 (!docs/pages/includes/tctl.mdx!)
 
 Before you get started you’ll need a running Teleport Enterprise or Cloud
-cluster on version 11.3.0 or greater.
+cluster on version `11.3.1` or greater.
 
 Login Rules only operate on SSO logins, so make sure you have
 configured an OIDC, SAML, or GitHub connector before you begin.
 Check the [Single Sign-On](../sso/) docs to learn how to set this up.
 
-## Step 1/4. Draft your Login Rule resource
+## Step 1/5. Configure a Teleport role with access to Login Rules objects
+
+First, ensure you are logged into Teleport as a user that has permissions
+to read and modify `login_rule` objects. The default `editor` role
+has access to this already, but in case you are using a more customized configuration,
+create a role called `loginrule-manager.yaml` with the following contents:
+
+(!docs/pages/includes/access-controls/loginrule-manager-role.mdx!)
+
+Create the role with `tctl`:
+
+```code
+$ tctl create loginrule-manager.yaml 
+role 'loginrule-manager' has been created
+```
+
+Add the role to your user. Replace `teleport-admin` with your user name:
+
+```code
+$ tctl users update --set-roles $(tctl get users/<Var name="teleport-admin"/> --format=json | \
+jq -r '.[].spec.roles | join(",")'), loginrule-manager <Var name="teleport-admin"/>
+User teleport-admin has been updated:
+	New roles: auditor,editor,access,loginrule-manager
+```
```

Comment:
Should we use the `add-role-to-user.mdx` partial here?

E.g.,

```markdown
(!docs/pages/includes/add-role-to-user.mdx role="loginrule-manager" !)

Reply:
Yes, @ptgott , I also applied this to the IDP example which is where I copied this from.